### PR TITLE
[14.0][MIG] sale_commission

### PR DESCRIPTION
**Pull request by mchristoph**

### CODE REVIEW

**Comment by Abranes** [comment]
@mchristoph please, follow the [migration process](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0#technical-method-to-migrate-a-module-from-130-to-140-branch)

**Comment by pedrobaeza** [comment]
Please cherry-pick #282

File: sale_commission/views/sale_commission_settlement_report.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+     <record id="action_report_settlement" model="ir.actions.report">
+        <field name="name">Settlement report</field>
+        <field name="model">sale.commission.settlement</field>
+        <field name="report_type">qweb-pdf</field>
+        <field name="report_name">sale_commission.report_settlement</field>
+        <field name="report_file">sale_commission.report_settlement</field>
+        <!-- todo needed? <field name="binding_model_id" ref="model_sale_commission_settlement"/>-->
```
Comment (fredzamoabg) [inline sale_commission/views/sale_commission_settlement_report.xml:9]: I think is needed

**Review by fredzamoabg — CHANGES_REQUESTED** [review]

**Comment by fredzamoabg** [comment]
@mchristoph could you please check comments on this PR? thanks

**Comment by pedrobaeza** [comment]
Superseded by #302

**Pull request closed without merging**